src/lib/stitching/exchange/v2/stitching.ts

```diff
@@ -546,6 +554,76 @@ export const exchangeStitchingEnvironment = ({
             return offerResult
           },
         },
+        submitOrderOffer: {
```

Comment:
Super minimal question: shouldn't it be **offer order** instead of **order offer**? (`submitOfferOrder`) I mostly see it expressed that way in our code and not the other way around. Plus, I think it's semantically an offer order, right? 😛

Reply:
I used Exchange's as the base, the name of the mutation there is `submitOrderWithOffer` since Offer is a sub thing under Order I thought that made sense but I can change it

Reply:
nvm, I see it's because we use commerceSubmitOrderWithOffer